build/github/build-and-publish-probes-for-operating-system/main.go

```diff
@@ -191,6 +210,48 @@ func getFalcoDrivers(dockerCli *docker.Client, FalcoVersionNames []string) ([]fa
 	return FalcoVersions, nil
 }
 
+func falcoVersionHasRelease(releases []*github.RepositoryRelease, falcoVersions []falcoVersion) bool {
+	for _, fv := range falcoVersions {
+		for _, r := range releases {
+			if strings.HasPrefix(fv.Driver, r.GetName()) {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+func kernelPackageHasBeenCompiled(releases []*github.RepositoryRelease, kernelPackage string) bool {
+	var released int
+	for _, r := range releases {
+		for _, a := range r.Assets {
+			if kernelPackageFromProbeName(a.GetName()) == kernelPackage {
+				released++
+				break
+			}
+		}
+	}
+
+	return released < len(releases)
+}
+
+func kernelPackageFromProbeName(probe string) string {
```

Comment:
nit: I think this would live nicely in `pkg/operatingsystem/kernel-package.go` as we have the inverse there. Though it seems `amazonlinux2` specific so might be worth under the `pkg/operating system/amazonlinux2` pkg. 
Bear in mind my comment here: https://github.com/thought-machine/falco-probes/pull/33#discussion_r684153126 i.e. we don't have a way to do this for Google Container OS (`cos`)